src/Symfony/Component/CssSelector/CssSelectorConverter.php

```diff
@@ -57,6 +64,10 @@ public function __construct(bool $html = true)
      */
     public function toXPath(string $cssExpr, string $prefix = 'descendant-or-self::')
     {
-        return $this->translator->cssToXPath($cssExpr, $prefix);
+        if ($this->cache[$prefix][$cssExpr] ?? false) {
+            return $this->cache[$prefix][$cssExpr];
+        }
+
+        return $this->cache[$prefix][$cssExpr] = $this->translator->cssToXPath($cssExpr, $prefix);
```

Comment:
would be slightly faster, we use this style elsewhere:
`return $this->cache[$prefix][$cssExpr] ?? $this->cache[$prefix][$cssExpr] = $this->translator->cssToXPath($cssExpr, $prefix);`

Reply:
Fixed thanks. I also added a bullet point in the CHANGELOG